src/Constants/SystemMessages.js

```diff
@@ -183,3 +183,12 @@ export const UPDATE_PUBLISAHBLE_POSITION_SUCCESS = 'This Publishable Position ha
 export const UPDATE_PUBLISAHBLE_POSITION__ERROR_TITLE = 'Publishable Position Error';
 export const UPDATE_PUBLISAHBLE_POSITION__ERROR = 'There was an issue attempting to save this Publishable Position. Please try again.';
 
+export const REMOVE_CYCLE_POSITION_SUCCESS_TITLE = 'Position Removed';
+export const REMOVE_CYCLE_POSITION_SUCCESS = 'Position Number has been removed from the cycle';
+export const REMOVE_CYCLE_POSITION_ERROR_TITLE = 'Position Delete Error';
+export const REMOVE_CYCLE_POSITION_ERROR = 'There was an issue attempting to remove this Position Number. Please try again.';
```

Comment:
```suggestion
export const REMOVE_CYCLE_POSITION_SUCCESS_TITLE = 'Cycle Position Removed';
export const REMOVE_CYCLE_POSITION_SUCCESS = 'Cycle Position has been removed.';
export const REMOVE_CYCLE_POSITION_ERROR_TITLE = 'Cycle Position Delete Error';
export const REMOVE_CYCLE_POSITION_ERROR = 'There was an issue attempting to remove this Cycle Position. Please try again.';
```

I know the ticket said Position Number, :/ but that sounds weird, tbh

Reply:
I was gonna suggest just `Position`, but when I ran it by Scott, he said `Cycle Position` made more sense